pkg/base/linuxutils/lsblk/lsblk.go

```diff
@@ -178,8 +174,9 @@ func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
 	vid := drive.VID
 	pid := drive.PID
 	for _, l := range lsblkOut {
-		if strings.EqualFold(l.Serial, sn) && strings.EqualFold(l.Vendor, vid) &&
-			strings.EqualFold(l.Model, pid) {
+		lvid := strings.TrimSpace(l.Vendor)
+		if strings.EqualFold(l.Serial, sn) && strings.EqualFold(lvid, vid) &&
+			strings.HasPrefix(l.Model, pid) {
```

Comment:
basically, it is ok. 
just noted that: if pid is null string(""),  then strings.HasPrefix will also return true.  but in our case,  pid seems to not able to be null. we may need re-look at this code in later release as well.